kong/plugins/acme/migrations/003_350_to_360.lua

```diff
@@ -1,41 +1,46 @@
 return {
     postgres = {
-      up = [[
-        DO $$
-        BEGIN
-          UPDATE plugins
-          SET config =
-            config
-                #- '{storage_config,redis}'
-
-            || jsonb_build_object(
-                'storage_config',
-                (config -> 'storage_config') - 'redis'
-                || jsonb_build_object(
-                  'redis',
-                  jsonb_build_object(
-                      'host', config #> '{storage_config, redis, host}',
-                      'port', config #> '{storage_config, redis, port}',
-                      'password', config #> '{storage_config, redis, auth}',
-                      'username', config #> '{storage_config, redis, username}',
-                      'ssl', config #> '{storage_config, redis, ssl}',
-                      'ssl_verify', config #> '{storage_config, redis, ssl_verify}',
-                      'server_name', config #> '{storage_config, redis, ssl_server_name}',
-                      'timeout', config #> '{storage_config, redis, timeout}',
-                      'database', config #> '{storage_config, redis, database}'
-                  ) || jsonb_build_object(
-                      'extra_options',
-                      jsonb_build_object(
-                          'scan_count', config #> '{storage_config, redis, scan_count}',
-                          'namespace', config #> '{storage_config, redis, namespace}'
-                      )
+      teardown = function(connector, _)
```

Comment:
I moved entire `up` section to `teardown`. The reason for that is for `acme` plugin we're only renaming `auth` to `password` inside `storage_config.redis`.
Apart from that we're adding `extra_options` to `storage_config.redis`. 

Now that I think of it, we could split it into addition of `extra_options` and password in the `up` section and the `teardown` section would contain removal of `auth` field. 

However I wonder what that brings us 🤔 I guess I'm missing some context on the `up` and `teardown` phases (i.e. how they are used).